client/src/locales/en.json

```diff
@@ -82,6 +89,11 @@
   "delete-form.email": "Your email",
   "delete-form.verify-message": "To verify, type {text} below",
   "delete-form.verify-text": "delete my account",
+  "email-notifications.save-settings-failure": "Changing notification frequency failed.",
+  "email-notifications.save-settings-success": "Email notification frequency saved.",
+  "email.notification.heading": "Email notifications frequency",
```

Comment:
Change "Email notifications frequency" to "Send summary report". Also "email.notification...." to "email.report... ".